Modules/Devices/src/Devices.Application/PushNotifications/Commands/UpdateDeviceRegistration/UpdateDeviceRegistrationValidator.cs

```diff
@@ -11,6 +11,8 @@ public UpdateDeviceRegistrationValidator()
     {
         RuleFor(dto => dto.Platform).In("fcm", "apns");
 
+        RuleFor(dto => dto.Environment).In("Development", "Production").When(dto => dto.Environment != null);
```

Comment:
Can't you instead do this:

```suggestion
        RuleFor(dto => dto.Environment).In("Development", "Production", null);
```